### PR TITLE
Add settings view

### LINKED ISSUE #27: ActitivtyRegistrator missing storage updates
The following part of ActitivtyRegistrator is out of date:

```javascript
// Create entry from information entered by user
const entry = {
  text: entryText,
  icon: route.params.icon,
  person: '', // TODO: link to value based on choie
  meaningful: importance,
  entertaining: enjoyment,
};

### PR DESCRIPTION
**Pull request by EMattfolk**
We have added the settings view. The navigation is still a bit wonky, but I am unsure if that is even possible to fix. It seems like even large corporations has the same problem.

### CODE REVIEW

File: src/language/en.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
+    "name": "English",
```
Comment (malin-mallan) [inline src/language/en.json]:
```suggestion
    "languageName": "English",
```
to specify a bit more. Name for me is a persons name. Or maybe just use "language".

File: src/language/sv.json [hunk 2]
```diff
@@ -1,8 +1,10 @@
 {
+    "name": "Svenska",
```
Comment (malin-mallan) [inline src/language/sv.json]:
```suggestion
    "languageName": "Svenska",
```
to specify a bit more. Name for me is a persons name. Or maybe just use "language".

**Review by malin-mallan — APPROVED** [review]
The layout looks good on both iOS and Android phones but I would like a titel on the top of the settingsside since there is nothing telling you were you are.

**Comment by Dolvur** [comment]
We added the alternative to change home icons on the settings which resulted in some new changes for iconlist

**Comment by malin-mallan** [comment]
The new settings locks good! I would still like to have a titel or something to indicate the navigation in the settings view. 

Also, if you enter settings from ex value, change to calender and then back to value you are still in settingsview and I don´t think that should happen.

Maybe you can use something like how it looks when entering the DatePicker

**Review by LukasJonsson99 — APPROVED** [review]
Looks good! Like Malin said would a title be a good thing to have.

**Comment by EMattfolk** [comment]
> The new settings locks good! I would still like to have a titel or something to indicate the navigation in the settings view.
> 
> Also, if you enter settings from ex value, change to calender and then back to value you are still in settingsview and I don´t think that should happen.
> 
> Maybe you can use something like how it looks when entering the DatePicker

I think these are good ideas, but I also think it is out of scope for this pull request. No other screens have a title as of now and adding one would probably mean all of them should have one. The thing with closing the settings should be possible, but the problem itself is probably very small.

**Pull request merged**